From 4bd289b612cf3cde769cf14f2e6e4d6cd78449df Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 11 Jul 2014 13:30:59 -0400
Subject: [redhat] Kconfig: add CONFIG_BOOTPARAM_HOTPLUG_CPU0=y on x86

Message-id: <1405085459-25379-1-git-send-email-prarit@redhat.com>
Patchwork-id: 85697
O-Subject: [RHEL7.1 PATCH BZ 902588] x86, add CONFIG_BOOTPARAM_HOTPLUG_CPU0=y
Bugzilla: 902588
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=902588

Build: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=7692928

Add CONFIG_BOOTPARAM_HOTPLUG_CPU0=y to allow CPU 0 hotplug.  This option
allows CPU 0, the boot cpu, to be hotplugged.

Successfully tested by me.  Note this patch requires mstowe's [RHEL7.1
PATCH 078/105] hotplug / x86: Fix online state in cpu0 debug interface,
for BZ 1114228.  He happened to post it just before I did :)

Cc: Don Zickus <dzickus@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Jarod Wilson <jwilson@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_BOOTPARAM_HOTPLUG_CPU0 b/redhat/configs/generic/x86_64/CONFIG_BOOTPARAM_HOTPLUG_CPU0
new file mode 100644
index 0000000..c5c9baf
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_BOOTPARAM_HOTPLUG_CPU0
@@ -0,0 +1 @@
+CONFIG_BOOTPARAM_HOTPLUG_CPU0=y
-- 
1.7.1